app/controllers/ops_controller/settings/label_tag_mapping.rb

```diff
@@ -150,14 +161,32 @@ def lt_map_set_new_form_vars
     session[:edit] = @edit
   end
 
+  def categories_for_select
+    categories = Classification.categories.sort_by(&:description)
+    categories.each_with_object([]) do |category, result|
+      next if category.read_only?
```

Comment:
Looks like you could do `Classification.categories.reject(&:read_only?).sort_by(&:description)` above

Reply:
Actually if all you want is the description you can just `.map(&:description)` and this is a one-liner